src/components/ContactsList/ContactRow.jsx

```diff
@@ -51,31 +52,34 @@ class ContactRow extends Component {
     const {
       contact,
       showModal,
+      divider,
       breakpoints: { isMobile }
     } = this.props
     const email = getPrimaryEmail(contact) || undefined
     const phone = getPrimaryPhone(contact) || undefined
     const cozyUrl = getPrimaryCozy(contact) || undefined
 
     return (
-      <div
-        className="contact"
+      <ListItem
+        className="u-c-pointer"
```

Comment:
The button prop should be used here. It takes care of the cursor styling.

Reply:
Oui mais j'ai eu des soucis de comportement avec button, notamment sur mobile, à cause du style "selected" qui ne s'affiche pas sur le bon item. Et là je n'ai plus trop le temps de regarder ça en profondeur.